ECMA2Yaml/IntellisenseFileGen/IntellisenseFileGenHelper.cs

```diff
@@ -499,6 +499,18 @@ private static void SpecialProcessElement(XElement ele)
                     });
                 }
 
+                // <summary><format type="text/markdown"><![CDATA[Describes the common properties that all features have.]]></format></summary> 
```

Comment:
what if there's markdown syntax in the CDATA, will it be processed as usual? For example, will
```
<summary><format type="text/markdown"><![CDATA[Describes the **common** properties that all features have.]]></format></summary> 
```
become
```
<summary>Describes the common properties that all features have.</summary> 
```
?

Reply:
yes, it i think formatEle.Value will escape CDATA automatically

Reply:
formatEle.Value will get value “Describes the common properties that all features have.”